try:
    a=1/0
except:
    print("erro")
finally:
    print('this will run at any condition')